src/routes/+layout.svelte

```diff
@@ -60,9 +60,8 @@
 		<div
 			on:mouseenter={() => ($isHoverHeader = true)}
 			on:mouseleave={() => ($isHoverHeader = false)}
-			class:bg-blue={$isHoverHeader}
 			class:text-white={$isHoverHeader}
-			class="bg-white fixed w-full top-0 z-10"
+			class="{$isHoverHeader ? 'bg-blue-500' : 'bg-white'} fixed w-full top-0 z-10"
```

Comment:
I think we should add condition when route is `/contact`, as I see in the design it's all `bg-blue-500`

Reply:
I think is trickier than that, because also the font needs to change and so on, for now its fine. but needs refinement later